### ISSUE TITLE
`rad sync --self` bug

### ISSUE THREAD

**Issue by cloudhead** [radicle-dev/radicle-cli#108]
It could be that not all refs are updated properly when `--self` is synced.
![image](https://user-images.githubusercontent.com/40774/168848353-9570b60f-7a68-4afa-85f8-4fcf70852a49.png)

**Comment by cloudhead** [comment]
Looks like the top-level ref isn't updated correctly on push:
```
> g ls-remote https://seed.alt-clients.radicle.xyz/hnrkqdpm9ub19oc8dccx44echy76hzfsezyio             # 130
7aadac2d0b905d854cc2d53d0298f465e0b3439f	refs/rad/id
76800334f44e9d0e66f1f69a5b6945d57de3728c	refs/remotes/hyn9diwfnytahjq8u3iw63h9jte1ydcatxax3saymwdxqu1zo645pe/rad/id
76800334f44e9d0e66f1f69a5b6945d57de3728c	refs/remotes/hyn9diwfnytahjq8u3iw63h9jte1ydcatxax3saymwdxqu1zo645pe/rad/self
92d7389fce745571ea4fed4c4dbfa4738c1ac28e	refs/remotes/hyn9diwfnytahjq8u3iw63h9jte1ydcatxax3saymwdxqu1zo645pe/rad/signed_refs
```

**Comment by cloudhead** [comment]
Fixed in radicle-client-services@2cf6beb0fdee88a0f7390b7346daa6e009bb980a
